examples/restrict-by-media-type/restrict_by_media_type.client.out

```diff
@@ -1,4 +1,4 @@
-# To invoke the service, use the following client.
+# To invoke the service, execute the following client.
```

Comment:
```suggestion
# To invoke the service, use the following cURL request.
```

Reply:
The word" execute" is more relevant than "use" here since the context is about a cURL command.

Reply:
Changed to "To invoke the service, execute the following cURL request.".